docs/include/conf/sponsorship/demographics.rst

```diff
@@ -11,8 +11,8 @@ For those people, they have a range of job titles.
 The most common job title is **Technical Writer**,
 but we have many other people who attend our conferences:
 
```

Comment:
```suggestion
The most common job title is **Technical Writer**,
but people with many other roles also attend our conferences:

```

Driveby not in the PR, soz. 
Also, shouldn't we delete this and use `docs/include/demographics.rst` ?